src/group/managed_group/mod.rs

```diff
@@ -384,11 +386,17 @@ impl<'a> ManagedGroup<'a> {
     // === Process messages ===
 
     /// Processes any incoming messages from the DS (MLSPlaintext &
-    /// MLSCiphertext) and triggers the corresponding callback functions
-    pub fn process_messages(&mut self, messages: Vec<MLSMessage>) -> Result<(), ManagedGroupError> {
+    /// MLSCiphertext) and triggers the corresponding callback functions.
+    /// Return a list of `GroupEvent` that contain the individual events that
+    /// occured whil processing messages.
+    pub fn process_messages(
+        &mut self,
+        messages: Vec<MLSMessage>,
+    ) -> Result<Vec<GroupEvent>, ManagedGroupError> {
```

Comment:
If managed group functions always return something like this, we should have a return type for it, i.e. a `Result` type in here with this signature to force consistency and make the API easier to understand.

Reply:
This is the only function that returns group events. I can add it if you insist, but the signature looked pretty clean to me.